tasks/request_status_for_job/request_status_for_job.py

```diff
@@ -55,53 +54,57 @@ def get_granule_status_entries_for_job(async_operation_id: str, db_connect_info:
                 granule_id,
                 orca_status.value AS status
             FROM
-                {DATABASE_SCHEMA_NAME}.orca_recoveryjob
-            INNER JOIN {DATABASE_SCHEMA_NAME}.orca_status ON orca_recoveryjob.status_id=orca_status.id
+                orca_recoveryjob
+            JOIN orca_status ON orca_recoveryjob.status_id=orca_status.id
             WHERE
                 job_id = %s
             """
 
     try:
         rows = database.single_query(sql, db_connect_info, (async_operation_id,))
-        result = database.result_to_json(rows)
-        return result
     except database.DbError as err:
         LOGGER.error(f"DbError: {str(err)}")
         raise DatabaseError(str(err))
+    result = database.result_to_json(rows)
+    return result
 
 
-def get_status_totals_for_job(async_operation_id: str, db_connect_info: Dict) -> Dict[str, int]:
+def get_status_totals_for_job(job_id: str, db_connect_info: Dict) -> Dict[str, int]:
+    # noinspection SpellCheckingInspection
     f"""
-    Gets the number of orca_recoveryjobs for the given {async_operation_id} for each possible status value.
+    Gets the number of orca_recoveryjobs for the given {job_id} for each possible status value.
 
     Args:
-        async_operation_id: The unique asyncOperationId of the recovery job to retrieve status for.
+        job_id: The unique id of the recovery job to retrieve status for.
         db_connect_info: The {database} defined db_connect_info.
 
     Returns: A dictionary with the following keys:
         'pending' (int)
         'success' (int)
         'failed' (int)
     """
+    # noinspection SpellCheckingInspection
     sql = f"""
-                SELECT 
-                    statuses.value,
-                    COUNT(granule_id)
-                FROM
-                    {DATABASE_SCHEMA_NAME}.orca_status as statuses
-                LEFT OUTER JOIN (SELECT * FROM {DATABASE_SCHEMA_NAME}.orca_recoveryjob WHERE {DATABASE_SCHEMA_NAME}.orca_recoveryjob.job_id=%s) AS jobs ON jobs.status_id=statuses.id
-                GROUP BY
-                    statuses.value
-        """
+            with granule_status_count AS (
+                SELECT status_id
+                    , count(*) as total
+                FROM orca_recoveryjob
+                WHERE job_id = %s
+                GROUP BY 1
```

Comment:
This is shorthanded, up to you if you would like more explicit.

Possibly `GROUP BY status_id`

Reply:
I'm a fan of stating things out.